### DISCUSSION TITLE
How to pass skipPackagePluginValidation in gym method in fastfile

### DISCUSSION THREAD

**Discussion by mukesh-mt** [fastlane/fastlane#20971] [Q&A] [answered]
I'm getting package validation failure when running builds with SPM Plugin, I want to skip package validation phase using -skipPackagePluginValidation argument in gym command.

**Comment by mukesh-mt** [comment, marked as answer]
Found it. I can use `xcargs` to pass build args.

**Reply by abin0992** [reply]
Hi @mukesh-mt, Can you please paste the exact code. I am using the following but CI fails

```
build_app(
  ...
  xcargs: "-skipPackagePluginValidation"
  ...
)
```

The error message from CI is as follows
`xcodebuild: error: invalid option '-skipPackagePluginValidation'`

**Reply by ast3150** [reply]
This worked fine for me:
`xcargs: '-skipPackagePluginValidation'`

Are you sure you're using Xcode 14?

**Reply by abin0992** [reply]
No, I was using Xcode 13.4 due to some other constraints. Using Xcode 14 fixed this.

**Comment by Ramandeep-S** [comment]
`xcargs: '-skipPackagePluginValidation'`
does not work for me 
my fastfile is in swift.
